topocalc/viewf.py

```diff
@@ -72,37 +133,25 @@ def viewf(dem, spacing, nangles=72, sin_slope=None, aspect=None):
         slope, aspect = gradient_d8(
             dem, dx=spacing, dy=spacing, aspect_rad=True)
         sin_slope = np.sin(slope)
+        cos_slope = np.cos(slope)
+        tan_slope = np.tan(slope)
```

Comment:
Minor: Could this be moved to the 2020 version?
This is only used there

Reply:
Agreed. I've moved common trig to both the 1990 and 2022 methods.